hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsConfigKeys.java

```diff
@@ -227,4 +227,24 @@ private HddsConfigKeys() {
   public static final String HDDS_SECURITY_CLIENT_SCM_CERTIFICATE_PROTOCOL_ACL =
       "hdds.security.client.scm.certificate.protocol.acl";
 
+  public static final String HDDS_DATANODE_HTTP_ENABLED_KEY =
+      "hdds.datanode.http.enabled";
+  public static final String HDDS_DATANODE_HTTP_BIND_HOST_KEY =
+      "hdds.datanode.http-bind-host";
+  public static final String HDDS_DATANODE_HTTPS_BIND_HOST_KEY =
+      "hdds.datanode.https-bind-host";
+  public static final String HDDS_DATANODE_HTTP_ADDRESS_KEY =
+      "hdds.datanode.http-address";
+  public static final String HDDS_DATANODE_HTTPS_ADDRESS_KEY =
+      "hdds.datanode.https-address";
+
+  public static final String HDDS_DATANODE_HTTP_BIND_HOST_DEFAULT = "0.0.0.0";
+  public static final int HDDS_DATANODE_HTTP_BIND_PORT_DEFAULT = 9882;
+  public static final int HDDS_DATANODE_HTTPS_BIND_PORT_DEFAULT = 9883;
+  public static final String
+      HDDS_DATANODE_HTTP_KERBEROS_PRINCIPAL_KEY =
+      "hdds.datanode.http.kerberos.principal";
```

Comment:
Can we just reuse the corresponding HDFS key here?

Reply:
@arp7 i think idea is to allow our hdds plugin to have separate identity.